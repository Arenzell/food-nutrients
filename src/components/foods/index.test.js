import React from 'react'
import Foods from './index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Foods updateNutrients={() => {}} />).toJSON()
  expect(tree).toMatchSnapshot()
})
