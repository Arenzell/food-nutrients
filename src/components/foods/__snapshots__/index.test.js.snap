// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="foodSelector"
  >
    <div
      className="ant-select-lg foodSelectorInput ant-select ant-select-enabled"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      onMouseOut={[Function]}
      onMouseUp={[Function]}
      onTouchStart={[Function]}
    >
      <div
        aria-autocomplete="list"
        aria-controls="test-uuid"
        aria-expanded={false}
        aria-haspopup="true"
        className="ant-select-selection
            ant-select-selection--single"
        data-gramm="false"
        onKeyDown={[Function]}
        role="combobox"
        tabIndex={0}
      >
        <div
          className="ant-select-selection__rendered"
        >
          <div
            className="ant-select-selection__placeholder"
            onClick={[Function]}
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "display": "block",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            + Add food
          </div>
          <div
            className="ant-select-search ant-select-search--inline"
            style={
              Object {
                "display": "none",
              }
            }
          >
            <div
              className="ant-select-search__field__wrap"
            >
              <input
                autoComplete="off"
                className="ant-select-search__field"
                onChange={[Function]}
                onKeyDown={[Function]}
                value=""
              />
              <span
                className="ant-select-search__field__mirror"
              >
                
                Â 
              </span>
            </div>
          </div>
        </div>
        <span
          className="ant-select-arrow"
          onClick={[Function]}
          style={
            Object {
              "WebkitUserSelect": "none",
              "userSelect": "none",
            }
          }
          unselectable="on"
        >
          <i
            aria-label="icon: down"
            className="anticon anticon-down ant-select-arrow-icon"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="down"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
              />
            </svg>
          </i>
        </span>
      </div>
    </div>
  </div>
  <div
    className="addFoods"
  >
    <i
      aria-label="icon: arrow-up"
      className="anticon anticon-arrow-up animate-flicker"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="arrow-up"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M868 545.5L536.1 163a31.96 31.96 0 0 0-48.3 0L156 545.5a7.97 7.97 0 0 0 6 13.2h81c4.6 0 9-2 12.1-5.5L474 300.9V864c0 4.4 3.6 8 8 8h60c4.4 0 8-3.6 8-8V300.9l218.9 252.3c3 3.5 7.4 5.5 12.1 5.5h81c6.8 0 10.5-8 6-13.2z"
        />
      </svg>
    </i>
  </div>
</div>
`;
